<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bittle</class>
 <widget class="QMainWindow" name="Bittle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bittle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="imageWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>434</width>
           <height>634</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="imageLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="offsetHandleBar">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="lblImageWidth">
          <property name="text">
           <string>Stride</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinImageStride">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblImageHeight">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinImageHeight">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>512</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>LSB First</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOuvrir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinImageStride</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Bittle</receiver>
   <slot>on_width_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinImageHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Bittle</receiver>
   <slot>on_height_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetHandleBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Bittle</receiver>
   <slot>on_offset_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Bittle</receiver>
   <slot>on_lsb_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_width_changed(int)</slot>
  <slot>on_height_changed(int)</slot>
  <slot>on_offset_changed(int)</slot>
  <slot>on_lsb_changed(int)</slot>
 </slots>
</ui>
